department-of-veterans-affairs/abd-vro
# MCP-2062: Make slack notifications more readable

## What was the problem?
1. slack exception notifications where somewhat hard to read and contained stacktraces
2. camel-bean-validator-starter introduces vulnerability
3. apparently, developers must be warned not to store PII/PHI


Associated tickets or Slack threads:

- [MCP-2062](https://amida.atlassian.net/browse/MCP-2062)

## How does this fix it?[^1]
1. make slack notifications more readable and remove stacktraces
2. removed camel-bean-validator-starter 
3. add a comment in AuditEvent to not store PII

## How to test this PR
- Execute a collectionId which causes an exception, which is almost all of them
- Inspect the slack channel


[^1]: [Pull-Requests guidelines](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Pull-Requests). If PR is significant, update [Current Software State](https://github.com/department-of-veterans-affairs/abd-vro/wiki/Current-Software-State) wiki page.
[^secrel]: To check if a PR will succeed in the SecRel workflow, [test PRs in the SecRel pipeline](https://github.com/department-of-veterans-affairs/abd-vro-internal/wiki/Secure-Release-process#to-test-prs-in-the-secrel-pipeline).
